core/src/main/java/org/springframework/security/access/intercept/AbstractSecurityInterceptor.java

```diff
@@ -201,17 +212,8 @@ protected InterceptorStatusToken beforeInvocation(Object object) {
 					"An Authentication object was not found in the SecurityContext"), object, attributes);
 		}
 		Authentication authenticated = authenticateIfRequired();
-		if (this.logger.isTraceEnabled()) {
```

Comment:
I see that you moved these to the adapter, which is understandable since they need the collection of config attributes.

Have you considered adding equivalent logs to `FilterInvocationAuthorizationManager`? It's helpful to see logs that explain the authorization context, especially reasons for authorization success of failure.